Gemfile

```diff
@@ -38,6 +38,9 @@ gem 'bootstrap_form'
 # Security libraries
 gem 'rack-attack', '~> 5.1'
 gem 'secure_headers', '~> 3.6', '>= 3.6.4'
+gem 'kaminari-mongoid'
+gem 'kaminari'
```

Comment:
Minor nitpicks: these aren't security libraries. It'd also be nice to have their versions set.